salt/kube-apiserver/apiserver.jinja

```diff
@@ -51,6 +51,4 @@ KUBE_API_ARGS="--advertise-address={{ grains['ip4_interfaces']['eth0'][0] }} \
                --oidc-client-id=caasp-cli \
                --oidc-ca-file={{ pillar['ssl']['ca_file'] }} \
                --oidc-username-claim=email \
-               --oidc-groups-claim=groups \
-               --storage-backend=etcd2 \
-               --storage-media-type=application/json"
```

Comment:
Don't we have to leave that around?

Reply:
I would prefer to keep as default, all migration was tested without --storage-media-type option. And origin PR was not including it https://github.com/kubic-project/salt/pull/206/files.This option was accidentally added in https://github.com/kubic-project/salt/pull/192/files#diff-7504d75a43d2c611b6f390b9c99a27d6R56